app/javascript/dashboard/components/widgets/AIAssistanceButton.vue

```diff
@@ -134,11 +135,25 @@ export default {
     },
   },
   mounted() {
+    document.addEventListener('keydown', this.onKeyDownHandler);
     if (!this.appIntegrations.length) {
       this.$store.dispatch('integrations/get');
     }
   },
+  destroyed() {
+    document.removeEventListener('keydown', this.onKeyDownHandler);
+  },
+
   methods: {
+    onKeyDownHandler(event) {
+      const { metaKey, ctrlKey, key } = event;
+      const shouldRevertTheContent =
+        (metaKey || ctrlKey) && key === 'z' && !!this.initialMessage;
```

Comment:
Can you take a look at KeyboardHelpers file `app/javascript/shared/helpers/KeyboardHelpers.js`? If possible, add the `metaKey || ctrlKey` condition there.